components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -195,6 +195,7 @@ class RewardsServiceImpl : public RewardsService,
   void GetReconcileTime(const GetReconcileTimeCallback& callback);
   void SetShortRetries(bool short_retries);
   void GetShortRetries(const GetShortRetriesCallback& callback);
+  void SetCurrentCountry(const std::string& current_country_);
```

Comment:
underscore at the end of variables should only be used for class member variables